app/services/data_request.rb

```diff
@@ -0,0 +1,73 @@
+require 'net/http'
+
+class DataRequest
+  attr_reader :data
+
+  # Set arguments here
+  def initialize(data)
+    @data = data

```

Comment:
It doesn't look like we're using the `data` argument anymore, so we can probably remove it to keep noise down in this class.

Reply:
You're right. We used it for testing and got errors when we removed it. The errors weren't apparent so we left it in to move forward. I believe it was something a long the lines of passing no argument when call expected one. Didn't know how to have it not expect one. Small thing I'm sure

Reply:
Cool, I can take a look if we still end up blocked on it, too